packages/wdio-cli/src/commands/run.ts

```diff
@@ -179,12 +179,17 @@ export async function handler(argv: RunCommandArguments) {
     if (isTSFile && !runsWithLoader && nodePath) {
         NODE_OPTIONS += ' --loader ts-node/esm/transpile-only --no-warnings'
         const localTSConfigPath = (
+            (
+                process.env.TS_NODE_PROJECT &&
+                path.resolve(process.cwd(), process.env.TS_NODE_PROJECT)
+            ) ||
             (
                 params.autoCompileOpts?.tsNodeOpts?.project &&
                 path.resolve(process.cwd(), params.autoCompileOpts?.tsNodeOpts?.project)
             ) ||
             path.join(path.dirname(wdioConf.fullPath), 'tsconfig.json')
         )
+        await fs.access(localTSConfigPath)
```

Comment:
Let's have a custom error message here

Reply:
I removed the error as it would be a breaking change and have added a console.log with a deprecation message Instead. I suggest that remove this when we release v9 :)

Reply:
sounds good